resources/views/Email/resetPassword.blade.php

```diff
@@ -0,0 +1,12 @@
+@component('mail::message')
+# Reset Password
+
+Reset or change your password.
+
+@component('mail::button', ['url' => 'http://localhost:3000/change-password?token='.$token])
```

Comment:
Em vez de usar uma url static, use variavel de ambiente APP_URL. Dessa forma, a gente apenas precisa mudar uma variavel para alterar todas as outras partes que precisam da APP_URL.

Reply:
Só essa view vai ter esse link. Nao vai ser usado em outros lugares.

Reply:
Entendi, o que eu quis diizer é que quando a gente for fazer deploy, por exemplo, a gente só muda a variavel de ambiente e ae as outras que usam a APP_URL, tipo o envio de mail, a url do avatar, etc. Serão alteradas facilitando nosso trabalho.

Reply:
Ah certo, você quis dizer o domínio da url.